fdb-record-layer-lucene/fdb-record-layer-lucene.gradle

```diff
@@ -26,21 +26,25 @@ if (!hasProperty('coreNotStrict')) {
 
 def coreProject = ":${ext.coreProjectName}"
 dependencies {
-    compile project(coreProject)
-    compile "org.apache.lucene:lucene-core:${luceneVersion}"
-    compile "org.apache.lucene:lucene-queryparser:${luceneVersion}"
-    compile "org.apache.lucene:lucene-analyzers-common:${luceneVersion}"
-    compile "org.apache.lucene:lucene-suggest:${luceneVersion}"
-    compile "org.apache.lucene:lucene-highlighter:${luceneVersion}"
-    testCompile project(path: coreProject, configuration: 'tests')
+    implementation project(coreProject)
```

Comment:
For similar reasons to `fdb-record-layer-icu`, I think this should be an `api` dependency

Reply:
changed to `api`